app/models/concerns/tristateable.rb

```diff
@@ -0,0 +1,31 @@
+###############################################################################
+## Tristateable
+## Reads a column, bypassing activerecord field accessors, so that we can
+## differentiate between nil values and boolean false (by default nils in
+## activerecord are converted to false).
+##
+## Naturally, booleans can only hold true/false, so these fields should be
+## created as strings.
+###############################################################################
+module Tristateable
```

Comment:
I think we should probably use a simpler comment style to be in line with the comment styles used in SM and RX apis. Its a nit-pick...